test/methods/template.js

```diff
@@ -22,6 +22,8 @@ test('$ can use newlines and tab indentations', testScriptStdout, () => $`echo.j
 	bar`, 'foo\nbar');
 test('$ can use newlines and space indentations', testScriptStdout, () => $`echo.js foo
   bar`, 'foo\nbar');
+test('$ can use escaped newlines and space indentations', testScriptStdout, () => $`echo.js foo\
+  bar`, 'foo\nbar');
```

Comment:
The test looks good. :+1: 

Could you please add the following tests as well?

```js
$`echo.js foo\
  bar` // two spaces, expects: 'foo\nbar'
```

```js
$`echo.js foo\
	bar` // inline tab (not spaces), expects: 'foo\nbar'
```

```js
$`echo.js foo\
\tbar` // escaped tab, expects: 'foo\tbar'
```

```js
$`echo.js foo\
\n\nbar` // escaped newlines, expects: 'foo\n\nbar'
```

Reply:
Okay. I also added one for the `foobar` case.

Reply:
Thanks!